sklearn/model_selection/_split.py

```diff
@@ -1769,6 +1769,43 @@ def __init__(self, *, n_splits=5, n_repeats=10, random_state=None):
             n_splits=n_splits,
         )
 
+    def split(self, X, y, groups=None):
+        """Generate indices to split data into training and test set.
```

Comment:
Just to double-check I guess you copied the docstring from `StratifiedKFold`?

Reply:
Yes, `RepeatedStratifiedKFold.split()` uses `StratifiedKFold.split()`.

This is similar to the previous code where `RepeatedStratifiedKFold` was using the split function from `_UnsupportedGroupCVMixin` due to inheritance and using super() class's split() function.

But because `RepeatedStratifiedKFold` is just similar to `StratifiedKFold` but with an extra `n_repeats` input parameter, it's better to use the doc string from StratifiedKFold.

Reply:
That's perfectly fine if you copied the docstring of `StratifiedKFold.split`. This seems like the right thing to do.